.github/workflows/build_wheels.yml

```diff
@@ -94,13 +94,11 @@ jobs:
           node-version: '17.x'
           registry-url: 'https://registry.npmjs.org'
       - name: Build and install dependencies
-        # There's a `git restore` in here because `make install-go-ci-dependencies` is actually messing up go.mod & go.sum.
         run: |
           pip install -U pip setuptools wheel twine
           make install-protoc-dependencies
           make build-ui
           git status
-          git restore go.mod go.sum
```

Comment:
These lines are okay to be deleted?

Reply:
We are not running `make install-go-ci-dependencies` now.